two_factor/templates/two_factor/twilio/sms_message.html

```diff
@@ -0,0 +1,3 @@
+{% load i18n %}
+{% trans sms_message %}
```

Comment:
This doesn't need to be a variable, if users want to override the default message they will override the template itself.

Reply:
done